### ISSUE TITLE
Packet injection with awus 1900 not working using kernel 5.10.0-kali2-amd64

### ISSUE THREAD

**Issue by edgarc96** [aircrack-ng/rtl8812au#824]

**Comment by Cleax21** [comment]
Having the same problem, Able to set the wifi adapter to monitor mode. but packet injection is something that isn't working.

Sitting on the same version as @edgarc96

**Comment by edgarc96** [comment]
I have read in other forums and chats that Parrot Os it is compatible with awus1900 chipset so ill try it today.

**Comment by Cleax21** [comment]
@edgarc96 did it work?

**Comment by astsam2** [comment]
AFAIK, the AWUS 1900 adapter is based on the RTL8814AU chipset. For this chipset check out another [driver](https://github.com/aircrack-ng/rtl8814au).

**Comment by edgarc96** [comment]
Yes, it worked, you have to download the drivers from rokland will redirect you to github page, after that you have to test the injection with aireplay-ng -9 (interface) ..... If you use aireplay-ng --deauth (interface) will not work or send any package

**Comment by edgarc96** [comment]
You have to use -9 instead of --test , sorry ..
